Foundation/Host.swift

```diff
@@ -28,6 +28,10 @@ open class Host: NSObject {
     internal var _names = [String]()
     internal var _addresses = [String]()
     
+#if os(Android)
+        static internal let NI_MAXHOST = 1025
```

Comment:
This line looks like it doesn't line up with the `static internal let` below - can you indent it to the left?

Reply:
Please don't forget to address this comment